src/App.tsx

```diff
@@ -77,6 +75,11 @@ export const App: React.FC = () => {
     setFilter(newFilter);
   }, []);
 
+  // eslint-disable-next-line react-hooks/rules-of-hooks
+  const handleSetIsError = useCallback((error: string | null) => {
```

Comment:
It would be better to create a type or enum for all errors instead of using strings because it allows for easier error management, prevents typos and improves code maintainability